syntax = "proto3";

package sample;

service SampleService {
  rpc SampleCall (SampleRequest) returns (SampleResponse) {}
}

message SampleRequest {
  string input = 1;
}

message SampleResponse {
  string output = 1;
}